RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerItemViewModel.cs

```diff
@@ -31,7 +31,8 @@ public class CompareByType : Comparer<CodeExplorerItemViewModel>
     {
```

Comment:
Does `NameWithSignature` include the access modifier for e.g. an `Implicit` public procedure? i.e. doesn't `Public Sub Zoo()` come before `Sub Foo()` if that's the case?